src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/DetailsVA.jsx

```diff
@@ -23,29 +24,41 @@ function formatHeader(appointment) {
   }
 }
 
-export default function DetailsVA({ appointment, facilityData }) {
+export default function DetailsVA({
+  appointment,
+  facilityData,
+  isVAOSServiceVAAppointments = false,
```

Comment:
we typically assign `useV2` as a boolean for VAOS Service api usage
```suggestion
  useV2 = false,
```